src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -293,6 +297,125 @@ def update_app_service_plan(instance, sku=None, number_of_workers=None,
         instance.admin_site_name = admin_site_name
     return instance
 
+def show_backup_configuration(resource_group_name, webapp_name, slot=None):
+    try:
+        return _generic_site_operation(resource_group_name, webapp_name,
+                                       'get_site_backup_configuration', slot)
+    except:
+        raise CLIError('Backup configuration not found')
+
+def list_backups(resource_group_name, webapp_name, slot=None):
+    return _generic_site_operation(resource_group_name, webapp_name, 'list_site_backups',
+                                   slot)
+
+def create_backup(resource_group_name, webapp_name, storage_account_url,
+                  db_name=None, db_type=None,
+                  db_connection_string=None, backup_name=None, slot=None):
+    client = web_client_factory()
+    if backup_name and backup_name.lower().endswith('.zip'):
+        backup_name = backup_name[:-4]
+    location = _get_location_from_webapp(client, resource_group_name, webapp_name)
+    db_setting = _create_db_setting(db_name, db_type, db_connection_string)
+    backup_request = BackupRequest(location, backup_request_name=backup_name,
+                                   storage_account_url=storage_account_url, databases=db_setting)
+    if slot:
+        return client.sites.backup_site_slot(resource_group_name, webapp_name, backup_request, slot)
+    else:
+        return client.sites.backup_site(resource_group_name, webapp_name, backup_request)
+
+def update_backup_schedule(resource_group_name, webapp_name, storage_account_url=None,
+                           frequency=None, keep_at_least_one_backup=False,
```

Comment:
If we're using action='store_true' for keep_at_least_one_backup, can the user ever set it to false? It seems like it would be set to None, and we would default to what was already in the schedule. How should we handle that situation?